### ISSUE TITLE
Example on the main page does has a bug!

### ISSUE THREAD

**Issue by erniejunior** [scikit-image/scikit-image-web#30]
Hi guys,
I just discovered your nice library. But apparently the showy demo script on the main page is broken. It should say
```python
from skimage import data, io, filter
```
and not 
```python
from skimage import data, io, filters
```
(one s too much on the filter).
Likewise on the line below it should be
```python
edges = filter.sobel(image)
```
At least I needed these changes to get it to work.

**Comment by JDWarner** [comment]
@erniejunior this is intended. We changed the name of the `filter` module to `filters` in 0.11, and deprecated `filter`. This is because Python has a builtin `filter` module, so someone who runs `from skimage import filter` shadows the builtin module.

If these snippets do not work for you, the most likely cause is an out of date installation; I suspect you are running 0.10 or earlier. In the current release, 0.11.x, `filter` will still work but warn the user it is deprecated. In future versions the `filter` submodule will be removed.

**Comment by erniejunior** [comment]
I installed scikit-image using pip (``pip install -U scikit-image``). I assumed that would install me the latest version.

**Comment by JDWarner** [comment]
I just double checked and this change was definitely included in the current stable 0.11 release (see the [release notes](http://scikit-image.org/docs/dev/release_notes_and_installation.html) in API changes).

So, either you are not running the latest stable release, or you have competing Python installations. In the latter case, your `pip` may have installed and/or updated scikit-image in a different installation of Python on your system than the one you are using when testing these scripts.

**Comment by jni** [comment]
as an aside, you can check your version of scikit-image as follows:

```python
import skimage
print(skimage.__version__)
```
